Makefile

```diff
@@ -70,3 +96,50 @@ else
 	@echo $(sign_skip_msg)
 endif
 	tar -czf $(appstore_package_name).tar.gz -C $(appstore_package_name)/../ $(app_name)
+
+# bin file definitions
+PHPUNIT=php -d zend.enable_gc=0  vendor/bin/phpunit
+PHP_CS_FIXER=php -d zend.enable_gc=0 vendor-bin/owncloud-codestyle/vendor/bin/php-cs-fixer
+
+.PHONY: test-php-codecheck
+test-php-codecheck:
+	$(occ) app:check-code $(app_name) -c private -c strong-comparison
+	$(occ) app:check-code $(app_name) -c deprecation
+
+.PHONY: test-php-lint
+test-php-lint:
```

Comment:
Do we actually want this lint stuff any more?
I think we will be doing the ownCloud code-style checks, and that includes parsing for syntax errors anyway?

Reply:
I have been discussing this with @patrickjahns already ... I think we need to come up with a final decision .... no idea if we get proper php5.6 syntax checks with all other tools we use ....

Reply:
@DeepDiver1975 
So what's the final take on php5.6 - will we drop it with EOL or not? 
Running phan on php7.0 , 7.1, 7.2 would automatically include linting, as the source is transformed into an AST by php itself. Which would fail if there are errors